scripts/synthdnn/synthdnn.py

```diff
@@ -184,12 +183,39 @@ def matmul_main(args):
         write_batch_file(sys.stdout, samples)
 
 
+def setup_report_subparser(subparsers):
+    report_parser = subparsers.add_parser(
+        "report", help="call with -h for information"
+    )
+    report_parser.add_argument(
+        "--subprogram_main", default=report_main, help=argparse.SUPPRESS
+    )
+    report_parser.add_argument(
+        "logs", nargs="*", help="path to benchdnn executable"
+    )
+    report_parser.add_argument(
+        "--scatter", action="append", help="x_idx,[y_idx],metric,[scale]"
+    )
+
+
+def report_main(args):
+    reports = ReportList(args.scatter)
```

Comment:
Defer
```suggestion
    from report.report_list import ReportList

    reports = ReportList(args.scatter)
```
to here to break a dependency on `matplotlib` when running `collect` mode.